bin/dev

```diff
@@ -8,7 +8,7 @@ const project = path.join(__dirname, '..', 'tsconfig.json');
 // In dev mode -> use ts-node and dev plugins
 process.env.NODE_ENV = 'development';
 
-require('ts-node').register({ project });
+require('ts-node').register({ project, swc: true });
```

Comment:
Any reason we wouldnt want `swc` on our 3PP template?

Reply:
none I can think of